avivator/src/utils.js

```diff
@@ -94,7 +94,7 @@ export async function createLoader(
     if (e instanceof UnsupportedBrowserError) {
       handleLoaderError(e.message);
     } else {
-      handleLoaderError(true);
+      handleLoaderError('');
```

Comment:
We are relying on the falsy-ness of an empty string in `LoaderError` now which is confusing IMO. I'd pass `null` here to be explicit that there is no error message, rather than an empty error message that is coerced to false in `LoaderError`.